twitwi/utils.py

```diff
@@ -23,8 +23,16 @@
 
 UTC_TIMEZONE = timezone("UTC")
 
+def safe_normalize_url(url):
+    try:
+        return partial(normalize_url, **CANONICAL_URL_KWARGS)(url)
+    except Exception:
+        return url # in case of error, return the original URL. Possibly not a valid URL, e.g. url containing double slashes
+
 custom_normalize_url = partial(normalize_url, **CANONICAL_URL_KWARGS)
 
+custom_safe_normalize_url = safe_normalize_url
```

Comment:
why creating a function then renaming it and only use the latter? why not just creating it immediately with the good name?